core/domain/user_services.py

```diff
@@ -494,19 +494,29 @@ def get_user_settings_from_username(username):
         return get_user_settings(user_model.id)
 
 
-def get_users_settings(user_ids):
+def get_users_settings(user_ids, strict=False):
     """Gets domain objects representing the settings for the given user_ids.
 
     Args:
         user_ids: list(str). The list of user_ids to get UserSettings
             domain objects for.
+        strict: bool. Whether to fail noisily if no user with the given ID
```

Comment:
... if one or more user IDs don't exist in the datastore. ...

Reply:
Done.